onlyswaps-verifier/src/signing.rs

```diff
@@ -99,10 +99,7 @@ pub fn create_message(params: SwapRequestParameters) -> Vec<u8> {
         params.amountOut,
         params.srcChainId,
         params.dstChainId,
-        params.verificationFee,
-        params.solverFee,
         params.nonce,
-        params.executed,
     )
         .abi_encode()
```

Comment:
For reference, this message is not unique per-router contract => two routers contracts on the same chain would result in the same message. This may allow re-using the signature to fulfill the swap using another router contract on the same chain (and, with the same signature scheme).